packages/fx-core/src/plugins/resource/frontend/index.ts

```diff
@@ -81,6 +81,13 @@ export class FrontendPlugin implements Plugin {
     );
   }
 
+  public async generateArmTemplates(ctx: PluginContext): Promise<TeamsFxResult> {
+    FrontendPlugin.setContext(ctx);
+    return this.runWithErrorHandling(ctx, TelemetryEvent.Deploy, () =>
```

Comment:
The telemetry event should not be deploy?

Reply:
Nice catch. Fixed.